src/geometries/TubeGeometry.js

```diff
@@ -165,9 +165,10 @@ function TubeBufferGeometry( path, tubularSegments, radius, radialSegments, clos
 
 			// vertex
 
-			vertex.x = P.x + radius * normal.x;
-			vertex.y = P.y + radius * normal.y;
-			vertex.z = P.z + radius * normal.z;
+			var r = radius instanceof THREE.Path ? radius.getPointAt( i / tubularSegments ).y : radius;
```

Comment:
BTW: Please no usage of the `THREE` namespace in the core 😉

Reply:
Yup! Fixed